src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestService/TestService.cs

```diff
@@ -74,6 +74,9 @@ protected override void OnShutdown()
 
         protected override void OnStart(string[] args)
         {
+            if (File.Exists(GetLogPath(ServiceName)))
```

Comment:
Is the exist check needed? I forget the behavior of Delete when the file doesn't exist; it throws?

Reply:
You're right - it isn't needed.